### DISCUSSION TITLE
Trading post watch list

### DISCUSSION THREAD

**Discussion by alazare619** [blish-hud/Blish-HUD#681] [Module Ideas]
Using the API would be possible to create a trading post watch list and query the prices and log them over a few days this exist outside with gw2bltc. Ideal ingame feature would be the following simplized features.

- Clicking name would put it on clipboard so can easily paste and serach BLTC
- Setup alert for notification if price drops below or raises above threshold
- Setup alert when quantity is below or above threshold
- Allow multiple lists i.e. Crafting Mats, Armor, or say potpuria

**Comment by Taschenbuch** [comment]
under development by Estreya
